Gamma_Analysis.py

```diff
@@ -43,13 +40,16 @@ def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
     return efficiency
 
 
-def emission_rate(net_area, efficiency, livetime):
+def emission_rate(net_area, livetime):
```

Comment:
It may need restructuring sooner given the Sea Salt reference essentially gives a specific count rate. Depends on what a common solution would be.